components/transport_profiles/lib/tasks/transport_profiles_tasks.rake

```diff
@@ -59,7 +59,7 @@ namespace :transport_profiles do
     end
     endpoint_references.keys.sort.each do |k|
       endpoint_results = ::TransportProfiles::WellKnownEndpoint.find_by_endpoint_key(k)
-      if endpoint_results.count < 1
+      if endpoint_results.any? == false
```

Comment:
Probably you can replace the `count <  1 ` with `.blank?`

Reply:
gonna switch it with .none? once I see everything else working